networking/src/main/kotlin/uk/govuk/app/networking/ui/component/ServiceNotRespondingMessage.kt

```diff
@@ -0,0 +1,105 @@
+package uk.govuk.app.networking.ui.component
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Icon
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import uk.govuk.app.design.ui.component.BodyBoldLabel
+import uk.govuk.app.design.ui.component.BodyRegularLabel
+import uk.govuk.app.design.ui.theme.GovUkTheme
+import uk.govuk.app.networking.R
+
+@Composable
+fun ServiceNotRespondingMessage(
```

Comment:
This is not final implementation and will be refined under  https://govukverify.atlassian.net/browse/GOVUKAPP-925